K2Bridge/Visitors/Aggregations/MetricAggregationVisitor.cs

```diff
@@ -23,6 +24,8 @@ public void Visit(AverageAggregation avgAggregation)
             EnsureClause.StringIsNotNullOrEmpty(avgAggregation.Field, avgAggregation.Field, ExceptionMessage);
 
             avgAggregation.KustoQL = $"{EncodeKustoField(avgAggregation.Key)}={KustoQLOperators.Avg}({EncodeKustoField(avgAggregation)})";
+
+            VisitedMetrics.Add(EncodeKustoField(avgAggregation.Key));
```

Comment:
Can we cut down on the repetitiveness of this?
I'm not sure how exactly, let me know what you think?

Reply:
I dislike also this line of code. I cannot do better at the moment because for some aggregation I need to get the key with metadata, and I don't have this information outside the visitor. But @tomconte is working on a solution that will perhaps solve this point. #teasing.

Reply:
Do we leave it like this for now for this PR? like we discussed we can transfer this to metadata table, but lets get it in first?

Reply:
Yes. We can improve this point later once everything will be transfered to metadata. At the moment, we can leave it like this.